src/Search/Type/Base.php

```diff
@@ -1,9 +1,9 @@
 <?php
-namespace Search\Type;
+namespace Search\Seach\Type;
 
 use Cake\ORM\Query;
 use Cake\Core\InstanceConfigTrait;
-use Search\Manager;
+use Search\Search\Manager;
```

Comment:
Well, it's the Search folder in the Search plugin, thus the double namespace is valid.
the vendor prefix would just be on top :)